frontend/src/styles/MomentView.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css?family=Teko');
+
+.accept-reject h1{
+    font-family: 'Teko';
+    font-size: 20px;
+}
+.caption-creator-input::placeholder{
+  font-family: 'Teko';
+  font-size: 23px;
+  letter-spacing: 0.8px;
+}
+.caption-creator-submit {
+   cursor: pointer;
+    width: 15%;
+    font-size: 28px;
+    border-radius: 3px;
+    border: none;
+    font-family: 'Teko';
+}
+.caption-creator-submit:active{
+    box-shadow: 1px 1.8px 1px #000000;
+    color:#1DE28F;
+}
+.caption-container .caption-content h1 a{
+  font-family: 'Teko';
+  font-size: 18px;
+}
+.caption-container h1 {
```

Comment:
again, should be defined for caption-container-accepted and caption-container-rejected